nf_core/create.py

```diff
@@ -348,6 +348,11 @@ def render_template(self):
             # Update the .nf-core.yml with linting configurations
             self.fix_linting()
 
+        log.debug("Dumping pipeline template yml to file")
+        if self.template_yaml:
+            with open(self.outdir / "pipeline_template.yml", "w") as fh:
+                yaml.safe_dump(self.template_yaml, fh)
```

Comment:
```suggestion
                yaml.safe_dump(self.template_yaml, fh)
                 run_prettier_on_file(fh.name)
```

Reply:
So this should only trigger when a `template_yml` is either supplied (which perhaps is silly) or someone customises something on the CLI.

Currently there are no tests which mock passing information in via the CLI prompts that I'm aware of. Happy to try adding them but not sure if it's best done as a unit-test or as part of one of the fuller tests defined in the GH actions?